lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -165,7 +167,23 @@ try {
         Exit-Json $result
     }
 
-    $principal = New-ScheduledTaskPrincipal -UserId "$user" -LogonType ServiceAccount
+    # Handle RunAs/RunLevel options for the task
+
+    if (!$store_password) {
```

Comment:
Are you able to swap this around, easier to see the first block is when `$store_password` is true and else is false